### PR TITLE
[PyTorch] Brain Mask Generator

### PR DESCRIPTION
**Pull request by dkurt**

### CODE REVIEW

File: modules/mo_pytorch/test/run_tests.py [hunk 1]
```diff
@@ -159,5 +160,38 @@ def forward(self, x):
...
+        from BrainMaGe.models.networks import fetch_model
+
+        weights = Path(BrainMaGe.__file__).parent / 'weights' / 'resunet_ma.pt'
+        pt_model = fetch_model(modelname="resunet", num_channels=1, num_classes=2, num_filters=16)
+        checkpoint = torch.load(weights, map_location=torch.device('cpu'))
+        pt_model.load_state_dict(checkpoint["model_state_dict"])
+        pt_model.eval()
+
+        # Get reference output
+        inp = torch.Tensor(np.random.standard_normal([1, 1, 128, 128, 128]).astype(np.float32))
```
Comment (likholat) [inline modules/mo_pytorch/test/run_tests.py:175]: Maybe `inp_shape = [1, 1, 128, 128, 128]` ? We use this list three times in test

**Review by likholat — APPROVED** [review]
